static/src/javascripts/projects/commercial/modules/prebid/utils.js

```diff
@@ -147,7 +143,7 @@ export const shouldIncludeMobileSticky = once(
         window.location.hash.indexOf('#mobile-sticky') !== -1 ||
         (config.get('switches.mobileStickyLeaderboard') &&
             isBreakpoint({ min: 'mobile', max: 'mobileLandscape' }) &&
-            isInNA() &&
+            isInUsRegion() &&
```

Comment:
Can we change this function's name to `isInNARegion()` for clarity? The function currently checks if the user is in the USA or Canada and since US != NA `isInNARegion()` seems more accurate.

Reply:
I think it gets clarified from having `Region` it its name likewise with `isInAuRegion` it checks for both 'AU' and 'NZ'